src/main/java/Thames/TaskList.java

```diff
@@ -0,0 +1,101 @@
+package Thames;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import Thames.task.ToDo;
+import Thames.task.Deadline;
+import Thames.task.Event;
+import Thames.task.Task;
+
+/**
+ * Array list of tasks.
+ */
+public class TaskList {
+    ArrayList<Task> taskList;
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Initializes the saved task list.
+     *
+     * @param sc Scanner file containing contents of saved task list.
+     */
+    public TaskList(Scanner sc) {
+        this.taskList = new ArrayList<>();
+        while (sc.hasNextLine()) {
+            String task = sc.nextLine();
+            String[] split = task.split(",");
```

Comment:
Should you use the singular form or plural for the names representing a collection of objects?